merky/test/exclude_annotation_nesting_test.py

```diff
@@ -0,0 +1,153 @@
+import collections
+
+from nose import tools
+from . import words
+import merky
+
+BASIC_SEQ = (words.SHEKELS, words.EUROS, words.EPEES)
+BASIC_SEQ_JSON = words.unify('[',
+                                '"', words.SHEKELS, '",',
+                                '"', words.EUROS, '",',
+                                '"', words.EPEES, '"',
+                             ']')
+BASIC_SEQ_HASH = '8129cb64b06e72be46eec5037eafc1583586325a'
+
+UNANNOTATED_NESTED_SEQ = (words.ANGSTROM, BASIC_SEQ, 213, -315)
+UNANNOTATED_NESTED_SEQ_JSON = words.unify('[',
```

Comment:
Your use of this variable and surrounding comments make me wonder if this constant name (prefix "UNANNOTATED_") is misleading and perhaps a relic of your initial copy-paste.  It looks like you're expecting it to *not* tokenize the `BASIC_SEQ_JSON`, given the JSON you construct here and the tests below.  However, that is only the case because the `BASIC_SEQ_JSON` *is* annotated.  So the variable names are throwing me a bit.